### ISSUE TITLE
SQLite support for DATE types with date mutator (__dates__, get_dates)

### ISSUE THREAD

**Issue by tsterker** [sdispater/orator#145]
Hello again :)

When I reference a column of type `DATE` in the `__dates__` array, I get an exception as `orator` seems to only expect `datetime` instances being listed there ([orator/orm/model.py#L2319..L2321](https://github.com/sdispater/orator/blob/0.9/orator/orm/model.py#L2319..L2321)):

```
  File ".../orator/orm/model.py", line 2321, in _get_attribute_value
    return self.as_datetime(value)
  File ".../orator/orm/model.py", line 2489, in as_datetime
    return pendulum.instance(value)
  File ".../pendulum/pendulum.py", line 198, in instance
    tz = dt.tzinfo or tz
AttributeError: 'datetime.date' object has no attribute 'tzinfo'
```

Is this on purpose or could `DATE` types also be supported?

**Update:**
I was using the `mysql` driver, which seems to automatically cast `DATE` types to `datetime.date`, e.g. as opposed to the `sqlite` driver, which seems to keep it as a string and makes the mutator work as expected.

**Comment by sdispater** [comment]
No, this is not on purpose.

Now that `pendulum`, which is used by `orator`, has a `Date` type we could use it to handle `DATE` columns.

I will check it out and get back to you.

**Comment by sdispater** [comment]
I just pushed a fix on the `0.9` branch that should address the issue.
